test/llvm-integration/LLVM.hs

```diff
@@ -1301,6 +1380,42 @@ defSymbols =
                         }
                 }
             )
+        ,
+            ( "Lbl'Unds'Int2Int'Pipe'-'-GT-Unds'"
+            , Symbol
+                { name = "Lbl'Unds'Int2Int'Pipe'-'-GT-Unds'"
+                , sortVars = []
+                , argSorts = [SortApp "SortWrappedInt" [], SortApp "SortInt" []]
```

Comment:
@geo2a I don't follow why the second argument has sort `Int` here; in the corresponding definition the second argument has sort `WrappedInt`? I might be missing something though.

Reply:
You're right, and it indeed makes the test I constructed to pass. We have been able to pin-point the problem to the `MAP.update` hook in the LLVM backend. I'll update the integration tests and also open an issue in the LLVm backend with the somewhat minimized definition and input.

Reply:
Remarkable timing; I had also just managed to build and get the same test to pass with the value sort as `WrappedInt`! Let's take a look at your new minimized example in our meeting this afternoon.